core/src/main/java/com/datastax/oss/driver/api/core/cql/PreparedStatement.java

```diff
@@ -58,6 +59,10 @@ public interface PreparedStatement {
   @NonNull
   ColumnDefinitions getVariableDefinitions();
 
+  /** Table name inferred from {@link PreparedStatement#getVariableDefinitions()}. */
+  @Nullable
+  CqlIdentifier getTable();
+
```

Comment:
I've renamed this to `inferTable()` for more clarity. `getKeyspace` and `getRoutingKeyspace` were already implemented. I've implemented `inferTable()` here because I wanted to go as specific as possible, without modifying superclasses I don't need to.